Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
So then we can have TimeProtocol, SoundProtocol etc that any VM can choose to implement in an extension, instead of some rigid hierarchy formed by inheritance of these "features".